codejail/django_integration.py

```diff
@@ -4,13 +4,14 @@
 
 """
 
+from __future__ import absolute_import
 from django.conf import settings as django_settings
 from django.core.exceptions import MiddlewareNotUsed
 
 from .integration import configure_from_settings
 
 
-class ConfigureCodeJailMiddleware(object):
+class ConfigureCodeJailMiddleware():
```

Comment:
I would rather you didn't remove object as a base class while we are still supporting Python 2.